kernel/src/grant.rs

```diff
@@ -721,22 +1006,22 @@ impl GrantRegionAllocator {
     /// initialized using the provided function.
     ///
     /// The provided function will be called exactly `n` times, and will be
-    /// passed the index it's initializing, from `0` through `NUM_ITEMS - 1`.
+    /// passed the index it's initializing, from `0` through `NUM_UPCALLS_ITEMS - 1`.
```

Comment:
Are these changes supposed to be here?

Reply:
nope, thats a find/replace mistake, sorry